src/StorageWriter/ClaimController.test.ts

```diff
@@ -0,0 +1,35 @@
+import { Messaging } from 'Messaging/Messaging'
+import { Db, Server } from 'mongodb'
+import * as Pino from 'pino'
+import { describe } from 'riteway'
+
+import { ClaimController } from './ClaimController'
+import { IPFS } from './IPFS'
+import { IPFSConfiguration } from './IPFSConfiguration'
+
+describe('Storage ClaimController', async (should: any) => {
+  const { assert } = should('')
+
+  const host = 'http://localhost'
+  const port = 3000
+  const server = new Server(host, port)
+  const IPFSConfiguration: IPFSConfiguration = {
+    ipfsUrl: '',
+  }
+
+  {
+    const claimController = new ClaimController(
+      Pino(),
+      new Db('poet', server),
+      new Messaging(),
+      new IPFS(IPFSConfiguration),
+    )
+
+    assert({
+      given: 'the new instance of ClaimController',
+      should: 'be an instance of ClaimController',
+      actual: claimController instanceof ClaimController,
+      expected: true,
```

Comment:
This just seems to be testing the 'new' keyword. What situation could ever cause instanceof to be false?

Reply:
This is code introduced by this PR https://github.com/poetapp/node/pull/230. Not in the scope of this PR. I agree this could be improved, IMO this test is really testing that it does not throw an error, so the `should` could be updated. But this should be discussed in a separate issue.

Reply:
Thanks. Have a look at using 'git mv' when renaming files if you are not familiar with it - it will prevent the renamed file from showing up as new code. I will say it's hard to get into the habit especially since renaming files is often done in IDEs, but makes PR review easier.

Reply:
Thanks, I am splitting a single module into 2 modules here so `git mv` would not work for this case.